system-container/root/exports/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "1.0",
+    "defaultValues": {
+        "OPTS": "",
+        "VAR_LIB_OPENSHIFT_INSTALLER" : "/var/lib/openshift-installer",
+        "VAR_LOG_OPENSHIFT_LOG": "/var/log/ansible.log",
+        "PLAYBOOK_FILE": "/usr/share/ansible/openshift-ansible/playbooks/byo/config.yml",
```

Comment:
As the symlink from `/usr/share/ansible` has been removed the `PLAYBOOK_FILE` should be a relative path (`playbooks/byo/config.yml`)